views/components/Graphs/BarGraph.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { 
+  VictoryBar, 
+  VictoryChart, 
+  VictoryAxis, 
+  VictoryTheme, 
+  VictoryStack } from 'victory';
+
+
+const BarGraph = ({ graph }) => {
+  const tickValues = graph.data.map((_, i) => (i))
+  const tickFormat = graph.data[0].map(({siteName}) => siteName)
+  const handlePercentage = (partialValue, totalValue) => (100 * partialValue) / totalValue
+ 
+  return(
+    <VictoryChart
+      domainPadding={20}
+      domain={{x: [0, 6]}}
+      theme={VictoryTheme.material}
+    >
+      <VictoryAxis
+        tickValues={tickValues}
+        tickFormat={tickFormat}
+      />
+      <VictoryAxis
+        dependentAxis
+        tickFormat={(x,_,a) =>  Math.round(handlePercentage(x, a[a.length - 1]))}
+      />
+    <VictoryStack>
+      {graph.data.map((data) => (
+        <VictoryBar data={data} x="siteName" y="count"/>
+      ))}
+    </VictoryStack>
```

Comment:
looks like the indentations off here, also I think we need a `key` on `VictoryBar` since we're using `map`?
```suggestion
      <VictoryStack>
        {graph.data.map((data) => (
          <VictoryBar key={data.siteName} data={data} x="siteName" y="count"/>
        ))}
      </VictoryStack>
```

Reply:
I had to use the index because of how the data has to be formatted for the Bar Graph. 
https://github.com/AMP-SCZ/dpdash/pull/91#discussion_r919419474

[a2afc3](https://github.com/AMP-SCZ/dpdash/pull/91/commits/a2afc3e7f7b1f052bfef361f91e567c91849655a)